python/cudf/cudf/__init__.py

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) 2018-2021, NVIDIA CORPORATION.
-from cudf.utils.gpu_utils import validate_setup  # isort:skip
+from cudf.utils.gpu_utils import validate_setup
```

Comment:
Just pointing out that this is an unrelated override that can also be removed as this line will not be resorted by 5.6.4